.ci/test.sh

```diff
@@ -159,10 +159,12 @@ elif [[ $TASK == "bdist" ]]; then
         ARCH=$(uname -m)
         if [[ $ARCH == "x86_64" ]]; then
             PLATFORM="manylinux1_x86_64"
+            # Make sure we can do both CPU and GPU; see tests/python_package_test/test_dual.py
+            export LIGHTGBM_TEST_DUAL_CPU_GPU=1
         else
             PLATFORM="manylinux2014_$ARCH"
```

Comment:
>  It even enables testing of the Integrated OpenCL build on arm64!

Can we try to integrate OpenCL into `aarch64` Linux wheels?
https://github.com/microsoft/LightGBM/blob/ab9236baa06c3f071acf6b45a771c2a09661f146/.vsts-ci.yml#L158
Or maybe you've already tried to do it?

Reply:
I have started to work on it. I had tested successfully with ubuntu 20.04, but then I noticed that we use CentOS in the CI. I hope to add a commit enabling aarch64 tomorrow.

Reply:
Sounds great!

Reply:
Enabled in 6a08e4f and c4e0533

Reply:
Unfortunately, `test_dual.py` fails with `lightgbm.basic.LightGBMError: No OpenCL device found` on `aarch64`.
https://dev.azure.com/lightgbm-ci/lightgbm-ci/_build/results?buildId=12929&view=logs&j=c2f9361f-3c13-57db-3206-cee89820d5e3&t=bf5b33f9-9072-549e-d8b1-cab79a1dd2a9&l=977

Reply:
Maybe [`gpu_platform_id`](https://lightgbm.readthedocs.io/en/latest/Parameters.html#gpu_platform_id) and [`gpu_device_id`](https://lightgbm.readthedocs.io/en/latest/Parameters.html#gpu_device_id) params can help?..

Reply:
This has been fixed in 0e25310. The issue was a wrong condition for detecting aarch64 (I was using the environment variable `ARCH` that was never set).